### DISCUSSION TITLE
Away from Campus due to Summer Break

### DISCUSSION THREAD

**Discussion by uffi29** [community/community#131661] [GitHub Education]
### Hi everyone,

I am a sophomore student and currently Summer Break is going on which means I am at home and my campus is in a different city. I am uploading a picture from my university's learning management system that there are no classes scheduled for today. But it is again giving review that your application cannot be accepted.
![Proof_Of_Leave](https://github.com/community/community/assets/175189965/f8c95384-15b1-428e-b55e-9e201173dd86)
![response](https://github.com/community/community/assets/175189965/f3a77578-6c58-4375-9f4d-c779ddb44a04)

**Comment by teddymillion** [comment]
yeah me to

**Comment by teddymillion** [comment]
what do we do

**Comment by davevad93** [comment]
Hi @uffi29 ,

As far as I know, applications from a location away from home are only allowed for distance learning students. If you are studying in a standard on campus program, I think you will have to wait until you finish your summer break and return to your home campus or close to it, in order to apply.

For more info, take a look at [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352).
